File: qc-core/src/main/java/com/eager/questioncloud/core/domain/review/event/DeletedReviewEvent.java
package com.eager.questioncloud.core.domain.review.event;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class DeletedReviewEvent {
    private Long questionId;
    private int rate;

    public static DeletedReviewEvent create(Long questionId, int rate) {
        return new DeletedReviewEvent(questionId, rate);
    }
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/review/event/DeletedReviewEvent.kt
package com.eager.questioncloud.core.domain.review.event

class DeletedReviewEvent(
    val questionId: Long,
    val rate: Int,
) {
    companion object {
        @JvmStatic
        fun create(questionId: Long, rate: Int): DeletedReviewEvent {
            return DeletedReviewEvent(questionId, rate)
        }
    }
}
